tests/acceptance/features/webUIFiles/options.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Set config options from phoenix config file
+
+  Background:
+    And user "user1" has been created with default attributes
+
+  Scenario: Set config to hide the search bar
+    Given the property "hideSearchBar" of "options" has been set to true in phoenix config file
+    And user "user1" has logged in using the webUI
```

Comment:
This can go to backgrounds section too...

Reply:
I just want to check both cases where config is changed before logging in and also when the user logged in